src/GraphQl/Resolver/Stage/ReadStage.php

```diff
@@ -172,4 +177,19 @@ private function getSubresource(string $rootClass, array $rootResolvedFields, st
             'collection' => true,
         ], $operationName);
     }
+
+    private function isNumericIndexArrayOfArray(array $array): bool
```

Comment:
I think it should be elsewhere, like in `Util/ArrayTrait` for instance.
Also, it should probably be split into two methods like `isSequentialArray` and `containsOnly` (maybe a more general method by using `gettype`).